sdcm/provision/aws/utils.py

```diff
@@ -259,7 +259,6 @@ def network_config_ipv6_workaround_script():
             MAC=`curl -s ${BASE_EC2_NETWORK_URL}`
             IPv6_CIDR=`curl -s ${BASE_EC2_NETWORK_URL}${MAC}/subnet-ipv6-cidr-blocks`
 
-            while ! systemctl status cloud-init.service | grep 'active (exited)'; do sleep 1; done
```

Comment:
Is it intentional line removal?

Reply:
Yes, it was moved to the beggining of the script